modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditor5EditorConfigContributor.java

```diff
@@ -25,10 +25,12 @@
  */
 @Component(
 	configurationPid = "com.liferay.frontend.editor.ckeditor.web.internal.configuration.CKEditor5Configuration",
-	property = "editor.name=ckeditor5_classic",
+	property = {
+		"editor.name=ckeditor5_balloon", "editor.name=ckeditor5_classic"
+	},
```

Comment:
For now, both versions of the editor have the same backed configuration, the same requirements from backend. Down the line, if they start to change, we may want to split this into multiple files.